yarn-project/key-store/src/test_key_store.ts

```diff
@@ -91,8 +95,11 @@ export class TestKeyStore implements KeyStore {
    */
   public async getMasterNullifierPublicKey(accountOrNpkMHash: AztecAddress | Fr): Promise<PublicKey> {
```

Comment:
Is it still used with an account or only with npkMHash?

I see that in noir the `getNullifierKeys` is only called with npk_m_hash so I would expect this to no longer be the case and it should be cleaned up.

Reply:
I still see accountOrNpkMHash all over the place so seems to not be complete

![image](https://github.com/AztecProtocol/aztec-packages/assets/13470840/f952219a-f8a0-4ff1-9b6a-abd8fcae4c95)

Reply:
Just go through the occurrences of `accountOrNpkMHash` and check if it's used with both. If it's just used by one then it's not supposed to be there (are there some occurrences when it's not the case? I couldn't find any.). It looks to me like a temporary hack you introduced to get the token note one mergeable.

Reply:
In what situations would we want to receive the nullifier keys with an address and not with `Npk_m` given that the keys can be rotated?

Reply:
yes, please make `accountOrNpkMHash` go away